src/masoniteorm/__init__.py

```diff
@@ -1,2 +1 @@
 from .models import Model
-from .factories.Factory import Factory
```

Comment:
why is this removed?

Reply:
The `Factory` class is only rewuired for Seeding as far as I can tell.

From a production environment perspective this is redundant
I added a [seeder] to the setup which will provide the Seeding capability if required.

I also noted that the Factory class should be imported from its package in the CHANGELOG in the  breaking changes section